### PR TITLE
fixes babel core error as reported in issue #79 

### PR DESCRIPTION
**Pull request by wk32**
This fixes the babel core error as reported in issue #79 by installing babel-preset-react-app as a dev dependency in /client

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -270,10 +268,10 @@ able to leave you command line in the root directory for the whole workshop.
...
 | :---: | :---: | :---: | :---: | :---: | :---: | :---: |
+| [<img src="https://avatars3.githubusercontent.com/u/7888225?v=4" width="100px;"/><br /><sub><b>WK</b></sub>](https://github.com/wk32)<br />[🐛](https://github.com/kentcdodds/testing-workshop/issues?q=author%3Awk32 "Bug reports") |
```
Comment (kentcdodds) [inline README.md]: This is actually generated. Please see the contributing guidelines in CONTRIBUTING.md to see how to add your name to the `.all-contributorsrc` file.

Comment (kentcdodds) [reply README.md]: No, there's a CLI that you can use which will edit the `.all-contributorsrc` file. Please see the instructions in CONTRIBUTING.md :)

Comment (wk32) [reply README.md]: I see. Is this the command:````yarn run add-contributor````? Think i ran it but did not include the file in the commit. Do i submit ````all-contributorsrc```` in a separate PR?

Comment (kentcdodds) [reply README.md]: It looks good now. Thanks!

**Pull request merged**